libs/execution_context/ebpf_protocol.h

```diff
@@ -39,6 +39,7 @@ typedef enum _ebpf_operation_id
     EBPF_OPERATION_BIND_MAP,
     EBPF_OPERATION_RING_BUFFER_MAP_QUERY_BUFFER,
     EBPF_OPERATION_RING_BUFFER_MAP_ASYNC_QUERY,
+    EBPF_OPERATION_RING_BUFFER_MAP_WRITE_DATA,
```

Comment:
nit: should we insert new values in the middle of the list? That will change the operation IDs for the rest of the enums.

Reply:
Since the ebpf API will always ship in conjunction with the ebpf core.sys, why is this a problem?